packages/reakit/src/Composite/__utils/getOppositeOrientation.ts

```diff
@@ -3,8 +3,9 @@ import { Orientation } from "./types";
 const map = {
   horizontal: "vertical",
   vertical: "horizontal",
+  both: "both",
 } as const;
 
 export function getOppositeOrientation(orientation?: Orientation) {
-  return orientation && map[orientation];
+  return orientation && (orientation === "both" ? "!both" : map[orientation]);
```

Comment:
I believe `"!both"` here is a mistake, no? Given we now have `orientation` on the map above, we can do 💭
```ts
return orientation && map[orientation];
```

Reply:
let's make necessary changes in this PR itself